packages/hierarchies-react/src/test/internal/TreeActions.test.ts

```diff
@@ -108,13 +109,15 @@ describe("TreeActions", () => {
       });
 
       const actions = createActions(model);
+
       actions.reloadTree();
 
       await waitFor(() => {
         expect(onModelChangedStub).to.be.calledOnce;
         const newModel = onModelChangedStub.firstCall.args[0];
         expect(newModel.rootNode.isLoading).to.be.true;
         expect(getHierarchyNode(newModel, "root-1")).to.not.be.undefined;
+        expect(provider.getNodes).to.be.calledOnce;
```

Comment:
why is this `getNodes` check needed? do we even need `await waitFor(() => { ... })` if we do `await actions.reloadTree().complete`?

Reply:
just removed getNodes.